crates/solidity/outputs/cargo/crate/src/lib.rs

```diff
@@ -1,18 +1,20 @@
 mod generated;
 
-mod _supress_binary_dependencies_ {
-    // Below are dependencies used by the binary `main.rs`, but not here.
-    // However, we need to add a fake usage below to supress Cargo warnings about unused dependencies.
-    // This is a known issue, and we should remove this hack once there is a better solution from Cargo.
-    // https://github.com/rust-lang/cargo/issues/1982
+// Below are dependencies used by the binary `main.rs`, but not here.
```

Comment:
I suggest undoing changes to this file, and keeping the comments inside the suppression module, as it improves readability when using folding.

Thanks for catching `thiserror` though! we should remove that.

Reply:
I separated the imports to be able to add `#[cfg(feature = "cli")]` to the entire suppression module instead of every individual import, should I still revert this?

Reply:
We can still have the `cfg` on top of the `mod` declaration. I believe it should still work.